azure-pipelines.yml

```diff
@@ -463,35 +466,23 @@ jobs:
     inputs:
       TargetPath: '$(Build.ArtifactStagingDirectory)/metadata'
       artifactName: metadata
-
-
   - task: DownloadPipelineArtifact@1
     displayName: 'Download PyPI Packages'
     inputs:
       TargetPath: '$(Build.ArtifactStagingDirectory)/pypi'
       artifactName: pypi
-
-
+  - task: UsePythonVersion@0
+    displayName: 'Use Python $(python.version)'
+    inputs:
+      versionSpec: '$(python.version)'
   - bash: |
       #!/usr/bin/env bash
-      
-      # Verify the pip wheels
-      
       set -ex
       
       CLI_VERSION=`cat $BUILD_ARTIFACTSTAGINGDIRECTORY/metadata/version`
-      PYPI_FILES=$(cd $BUILD_ARTIFACTSTAGINGDIRECTORY/pypi; pwd)
-      
-      echo "== Testing pip install on Python 3.9 =="
-      docker run \
-        --rm -v $PYPI_FILES:/mnt/pypi mcr.microsoft.com/mirror/docker/library/python:3.9-slim \
-        /bin/bash -c "cd /mnt/pypi && ls && pip install --find-links ./ azure_cli-$CLI_VERSION*whl && az self-test && az --version && sleep 5"
-      
-      echo "== Testing pip install on Python 3.11 =="
-      docker run \
-        --rm -v $PYPI_FILES:/mnt/pypi mcr.microsoft.com/mirror/docker/library/python:3.11-slim \
```

Comment:
MAR does not provide newer Python docker image anymore, use local Python instead.